influxdb_client_3/write_client/client/write/dataframe_serializer.py

```diff
@@ -13,16 +13,24 @@
 
 logger = logging.getLogger('influxdb_client.client.write.dataframe_serializer')
 
+try:
+    from ...extras import pd
+
+    def _not_nan(x):
+        return not pd.isna(x)
+
+except ImportError:
+    pd = None
+
+    def _not_nan(x):
+        return x == x
+
```

Comment:
This can be simplify to because the pandas should be present:

```suggestion
def _not_nan(x):
    from ...extras import pd
    return not pd.isna(x)

```

Reply:
done